libcxx/modules/std/stacktrace.inc

```diff
@@ -8,7 +8,9 @@
 //===----------------------------------------------------------------------===//
 
 export namespace std {
-#if 0
+// TODO(stacktrace23): update this when stacktrace is taken out of experimental
+#if 0 //_LIBCPP_STD_VER >= 23
```

Comment:
As everywhere else I think this needs to be guarded via the "experimental" macro you should implement.